ansible/configs/controller-migration-casc/files/repos_template.j2

```diff
@@ -0,0 +1,38 @@
+[rhel-7-server-rpms]
+name=Red Hat Enterprise Linux 7
```

Comment:
rhel7? Seems a bit old, is that intended?

Reply:
not used